src/content/docs/hooks/usedimensions/BasicExample.jsx

```diff
@@ -0,0 +1,38 @@
+import React, {useRef} from 'react';
+import {useDimensions} from 'webrix/hooks';
+import './BasicExample.scss';
+
+const getcolor = (height) => {
```

Comment:
Let's apply the color/bg in the SCSS file, and just add classNames here.
Also, consider using an object literal instead of `if else`, since it's a bit more readable:
```
const getClassName = height => Object.entries({
    s: height < 40,
    m: height >= 40 && height < 80,
    l: height >= 80 && height < 130,
    xl: height >= 130 && height < 170,
}).reduce((acc, [cls, inRange]) => inRange ? cls : acc, '');
```